t/10-jobs.t

```diff
@@ -679,7 +679,6 @@ subtest 'carry over, including soft-fails' => sub {
 
             # Defaults (retry with delay)
             $job->settings->search({key => '_TRIGGER_JOB_DONE_DELAY'})->delete;
-            $job->settings->search({key => '_TRIGGER_JOB_DONE_RETRIES'})->delete;
```

Comment:
why?

Reply:
https://github.com/os-autoinst/openQA/pull/6415#pullrequestreview-2830835533 states that you should reduce the default which this PR so far does not do. And it states that you should add a test to test the exhaustion of retries. Instead what you do in the test that states that it would test the defaults is now actually not testing defaults anymore

Reply:
Correct. I oversaw that this was checking the default, so I thought it is simplier. I extracted to a subtest.